packages/client/src/scenes/loadWorldScene.ts

```diff
@@ -49,6 +50,14 @@ export class LoadWorldScene extends Phaser.Scene {
     background.setDisplaySize(SCREEN_WIDTH, SCREEN_HEIGHT);
     background.setDepth(-10);
 
+    // Add frame
+    const overlayImage = this.add.image(SCREEN_WIDTH /2, SCREEN_HEIGHT / 2, 'frame');
+    overlayImage.setScrollFactor(0);
+    console.log('FrameScene created');
+    // background.setOrigin(0, 0);
+    // background.setDisplaySize(SCREEN_WIDTH, SCREEN_HEIGHT);
+    // background.setDepth(-10);
+
     //loadWorld().then(() => {
```

Comment:
Could we remove this code that is just commented out? It just adds noise to the codebase